xbmc/media/MediaType.cpp

```diff
@@ -75,6 +75,14 @@ MediaType MediaTypes::ToPlural(const MediaType &mediaType)
   return mediaTypeIt->second.plural;
 }
 
+bool MediaTypes::IsCollection(const MediaType &mediaType)
```

Comment:
@mkortstiege: `MediaTypeVideoCollection is something else (i.e. a "movie set").

But there already is a `MediaTypes::IsContainer()` method that you can use.